### DISCUSSION TITLE
多级表头如何导出csv

### DISCUSSION THREAD

**Discussion by niceboyat** [layui/layui#2352] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

2.9.18

### 问题描述

多级表头如何导出csv?我尝试好像只能支持一级表头。

### 业务代码

```auto
无
```

### 浏览器

chrome

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
多级表头当前只支持导出一级表头。

**Reply by niceboyat** [reply]
> 多级表头当前只支持导出一级表头。

后续考虑加这个功能吗

**Reply by LSL1618** [reply]
> > 多级表头当前只支持导出一级表头。
> 
> 后续考虑加这个功能吗

推荐使用第三方插件实现
[https://github.com/exceljs/exceljs](https://github.com/exceljs/exceljs)
[https://github.com/rockboom/SheetJS-docs-zh-CN](https://github.com/rockboom/SheetJS-docs-zh-CN)
